src/index.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * Marlyn Lopez, Liqing Li
+ * 6/10/2022
+ * Weather report pair project
+ * Finished all wave 1-wave 6 features development.
+ * Note: plus, minus, get real time temp, city selection bar and input bar buttons
+ * are set as position relative(parent) and absolute(child) in css file. 
+ * So as window size changes these buttons move around, we know it's not the best implementation.
+ * We were trying to focus on practicing html event handling with javascript for this project.
+ * 
+ */
+
+/* Wave2 
+1. increase and decrease temperature on click 
+2. temperature # change text color and background color accordingly
+3. temperature range changes garden emoji at bottom
+*/
+
+const state = {
+  tempIncrement: 77,
+};
+
+//check background color when click temperature +/- button
+const changeBackColorPlusMinusBtn = () => {
+  const gardenEmoji = document.querySelector('#temp-emoji');
+  if (state.tempIncrement >= 80) {
+    document.body.style.backgroundColor = 'red';
```

Comment:
Style: It would be a little more maintainable if this instead set a class (something like `hot`) and had CSS choose the color.  

That way all of your styles would be in a single place.